src/Accelerators/NNPA/Dialect/ZHigh/ZHighOps/Stick/ZHighStick.td

```diff
@@ -80,4 +80,42 @@ def ReplaceONNXLeakyReluPattern: Pat<
    (SameLayout $X, $stickout)]
 >;
 
+// Since zDNN does not support Sqrt(X), we calculate it by using zDNN-supported
+// operations, i.e. Exp and Log.
+// Formulas: `sqrt(X) = exp(log(x)/2) = exp(0.5 * log(x))`
+def ReplaceONNXSqrtPattern: Pat<
+  (ZHighStickOp:$stick (ONNXSqrtOp (ZHighUnstickOp $X)), $layout),
+  (ZHighExpOp (ZHighMulOp (ZHighLogOp $X, (returnType $X)),
+                          (ZHighStickOp (GetConstantOfType<"0.5"> $X), $layout),
+                          (returnType $X))),
+  [(IsStaticShapeTensor $X), (SameLayout $X, $stick)]
```

Comment:
It is because of the second operand of `ZHighMulOp` that is a constant of `0.5`. We create a constant tensor `(GetConstantOfType<"0.5"> $X)` by using X's static shape. Then ZHighStickOp of the constant can be folded at compile time. Otherwise, there is overhead of stickification.

Reply:
got it, tx.